### ISSUE TITLE
include options fails if array length > 1

### ISSUE THREAD

**Issue by m0rtadelo** [m0rtadelo/recursive-readdir-async#3]
The include options fails if has more than one item (array length > 1).

**Comment by jmaferreira** [comment]
The same is happening to me.

**Comment by jmaferreira** [comment]
Here's a fix. Replace the following function with the following code:

```javascript

function onlyInclude() {
  for (let i = 0; i < list.length; i++) {
    let removeItem = true;
    for (let j = 0; j < settings.include.length; j++) {
      let item = list[i];
      if (settings.mode === TREE && item.isDirectory && item.content) continue;
      if (item.fullname.indexOf(settings.include[j]) > 0) {
        removeItem = false;
      }
    }
    if (removeItem) list.splice(i, 1);
  }
  return list;
}

```

**Comment by jmaferreira** [comment]
@m0rtadelo Any change you will release a fix soon?

**Comment by m0rtadelo** [comment]
Sorry. I will fix it as soon is possible. Thank you.

**Comment by m0rtadelo** [comment]
Miguel i have check your pull request and breaks unit tests (check travis build https://travis-ci.org/github/m0rtadelo/recursive-readdir-async/builds/709577283). I cannot accept your pull request.
Obrigado

**Comment by jmaferreira** [comment]
I understand... but please advise... all of those fails are due to my change? I've only changed one function...

**Comment by jmaferreira** [comment]
You're right.. its buggy. Sorry.

**Comment by m0rtadelo** [comment]
Don't worry. This issue has been fixed in #10 and published as release 1.1.7

**Comment by jmaferreira** [comment]
Dude, you're best! Cheers.
